client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -962,6 +966,35 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 		/>
 	);
 
+	const bigSkyButtonEventProperties = {
+		is_big_sky_eligible: isBigSkyEligible,
+		// is_filter_included_with_plan_enabled: true/false,
+		// preselected_filters: ??,
+		// selected_filters: ??,
+		// {filter} ??
+	};
+	const bigSkyButtons = (
```

Comment:
Let's gate the feature like [here](https://github.com/Automattic/wp-calypso/pull/97004/files#diff-a539d272b24cf98cb319fb27637a06c9faf52cfea40d0af87cf1b7c2fec0915cR122) and add the common properties in the follow-up PR!

Reply:
Updated here e47aad3afe330247945a0fa0cea6141480b05cbc